esrally/telemetry.py

```diff
@@ -91,6 +93,8 @@ def on_benchmark_start(self):
         for device in self.devices:
             if self._enabled(device):
                 device.on_benchmark_start()
+            if self._excluded_on_serverless(device):
```

Comment:
Don't we want this before calling `on_benchmark_start()`, and to also `continue`?

Reply:
Oh, of course, thanks. I was so focused on the message that I forgot the functionality. :sweat_smile: 

I'll leave it like that for now to make sure my tests catch it.